apps/web/src/routes/AllView/AllView.jsx

```diff
@@ -102,13 +102,13 @@ export default function AllViews () {
     return widgetsSorted.map(widget => {
       const title = widget.title
       const url = widgetUrl({ widget, rootPath, groupSlug: routeParams.groupSlug, context: 'group' })
-      const type = widgetTypeResolver({ widget })
+      const type = widgetTypeInferrer({ widget })
       const capitalizedType = type.charAt(0).toUpperCase() + type.slice(1)
```

Comment:
Resolve in presenter as typeTitle and same below viewTitle... Then user here as widget.typeTitle and widget.viewTitle...